src/detect-engine.c

```diff
@@ -4092,7 +4093,7 @@ int DetectEngineReload(const SCInstance *suri)
     old_de_ctx = DetectEngineGetCurrent();
     if (old_de_ctx == NULL)
         return -1;
-    SCLogDebug("get ref to old_de_ctx %p", old_de_ctx);
```

Comment:
Oops. Deleted this by mistake. Fixing in next rev

Reply:
ah I see this now, nm :)